src/main/kotlin/com/numberone/daepiro/domain/baseentity/PrimaryKeyEntity.kt

```diff
@@ -20,7 +20,7 @@ abstract class PrimaryKeyEntity {
 
     @CreatedDate
     @Column(nullable = false, updatable = false)
-    val createdAt: LocalDateTime = LocalDateTime.now()
+    var createdAt: LocalDateTime = LocalDateTime.now()
```

Comment:
<code> Cannot set property createdAt because no setter, no wither and it's not part of the persistence constructor public com.numberone.daepiro.domain.user.entity.UserEntity</code>
위와 같은 에러가 유저생성시 발생하더라구요. 아마 createdAt에 할당하는 값도 단순히 객체 생성시 할당되는게 아니라 jpa가 내부적으로 후주입해서 그런것은 아닐까 생각이 드네요

Reply:
아 맞는 것 같네요.
jpa 에서 setter 로 주입하려고 하는데 불변이라 안되는 상황이군요..
하나 배워갑니다!